### ISSUE TITLE
Update part of screen

### ISSUE THREAD

**Issue by MrVine** [abiosoft/ishell#104]
Is it possible to update concrete line of `ishell`?

I creating some application, which will return `results []*Result` slice to user. In my application hard background work should be done for each element of slice, so I want to notify user about current progress by sending new state of `*Result` item over `chan *Result` channel. 

So, is it possible to update only concrete line of `ishell`, without re-writing all content of `results` slice?

**Comment by abiosoft** [comment]
I don't fully understand what you are trying to do. But sounds like you can make use of the progress bar and simply set your progress like this https://github.com/abiosoft/ishell/blob/master/example/main.go#L144.

**Comment by abiosoft** [comment]
No, you cannot do that yet. I actually have it listed in the [roadmap](https://github.com/abiosoft/ishell#roadmap-in-no-particular-order), may prioritise it.

**Comment by MrVine** [comment]
Ok, thanks
